libnmstate/ifaces/ifaces.py

```diff
@@ -198,6 +193,32 @@ def _apply_copy_mac_from(self):
             else:
                 iface.apply_copy_mac_from(port_iface.mac)
 
+    def _validate_copy_mac_from_iface_is_in_port(self, iface):
+        iface_port = self._port_for_interface(iface)
+        if iface.copy_mac_from not in iface_port:
+            raise NmstateValueError(
+                f"The interface {iface.name} is holding invalid "
+                f"{Interface.COPY_MAC_FROM} property "
+                f"as {iface.copy_mac_from} is not in the port "
+                f"list: {iface.port}"
+            )
+
+    def _port_for_interface(self, iface):
+        if iface.type == InterfaceType.OVS_INTERFACE:
+            ovs_bridges = (
+                x
+                for x in self.all_ifaces()
```

Comment:
Please use `self.all_user_space_ifaces` to search OVS bridge only.

Reply:
Thanks! [Updated](https://github.com/nmstate/nmstate/compare/79e191b615bd5aa69cdc4e7a30d866e6b055d074..d8396b8a764a10da9cfec0f61b58b624e6fdce3f)